### PR TITLE
DUOS-1822[risk=med] Enable MDV for Prod, Updated Env Conditionals

### PR DESCRIPTION
**Pull request by quazi-h**
[DUOS-1822](https://broadworkbench.atlassian.net/browse/DUOS-1822)

- Update all nav links in routes to apply to all environments.
- Removed conditional logic that considers the environment for routes to the console pages.
- Updated the app to exclusively use the new DUOS header.
- Updated links in the new and old header code to point to the new consoles.
- Updated utils to point to new consoles.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by quazi-h** [comment]
Open Questions:
- Should my changes include switching over to the new header and nav bar for prod?
- Unsure if I replaced the isRendered arguments correctly in SigningOfficialDataSubmitters and SigningOfficialConsole.
- Wasn't sure where to add tests/which tests to update.
- auth.cy.js is currently failing:
```
  1) Authentication
       Background Sign-in Admin:
     Error: Must pass in a JSON object containing the Google auth settings.
      at GoogleAuth.fromJSON (http://localhost:3000/__cypress/tests?p=cypress/support/e2e.js:48698:19)
      at Context.eval (http://localhost:3000/__cypress/tests?p=cypress/support/e2e.js:41585:23)
```

**Comment by rushtong** [comment]
> Open Questions:
> 
> * Should my changes include switching over to the new header and nav bar for prod?

No, this should not cover the navigation. That is covered in https://broadworkbench.atlassian.net/browse/DUOS-1887. The new navigation is dependent on MDV, so this has to go first.

File: src/App.js [hunk 1]
```diff
@@ -73,16 +73,12 @@ function App() {
...
-  const header = env !== 'prod'
-    ? h(DuosHeader, { onSignOut: signOut })
-    : h(DuosHeaderOld, { onSignOut: signOut });
 
   return (
     div({ className: 'body' }, [
       div({ className: 'wrap' }, [
         div({ className: 'main' }, [
-          header,
+          h(DuosHeader, { onSignOut: signOut }),
```
Comment (rushtong) [inline src/App.js]: These changes are related to DUOS-1887 and outside of the scope of the linked ticket.

File: src/pages/AdminConsole.js [hunk 2]
```diff
@@ -163,7 +163,7 @@ class AdminConsole extends Component {
             div({ className: 'row fsi-row-lg-level fsi-row-md-level no-margin' }, [
               // TODO: `admin_manage_access` will be fully deprecated with MDV
               div({
-                isRendered: (this.props.env === 'prod'),
+                isRendered: this.state.showAddUserModal,
```
Comment (rushtong) [inline src/pages/AdminConsole.js]: This div should be removed altogether as it is referencing an outdated page, Manage DARs.

File: src/pages/AdminConsole.js [hunk 3]
```diff
@@ -177,7 +177,7 @@ class AdminConsole extends Component {
                 }),
               ]),
               div({
-                isRendered: (this.props.env !== 'prod'),
+                isRendered: this.state.showAddUserModal,
```
Comment (rushtong) [inline src/pages/AdminConsole.js]: This div should always be enabled now. It is essentially a replacement for the previous div.

File: src/pages/AdminConsole.js [hunk 4]
```diff
@@ -217,7 +217,7 @@ class AdminConsole extends Component {
               ]),
               div({
                 // TODO: We won't need this box when MDV is complete
-                isRendered: (this.state.env === 'prod'),
+                isRendered: this.state.showAddUserModal,
```
Comment (rushtong) [inline src/pages/AdminConsole.js]: This div can go away altogether, it is replaced by the div on line 179

**Comment by rushtong** [comment]
> Open Questions:
> 
> * Unsure if I replaced the isRendered arguments correctly in SigningOfficialDataSubmitters and SigningOfficialConsole.

Signing official changes are related to navigation, not MDV, so please revert those.

**Review by rushtong — CHANGES_REQUESTED** [review]
This PR mixes some navigation changes in with MDV, so let's try to split them back out so they can be separately addressed. One issue I see is that we've lost some CSS for the old header - likely in a previous PR. Once I back out the new header nav, the UI looks like the attached screenshot. We'll need to get that back before this goes forward.

Screen Shot 2022-07-12 at 6 44 07 AM

File: src/App.js (hunk 1 above)
Comment (quazi-h) [reply src/App.js]: Backed these changes out.

**Review by rushtong — APPROVED** [review]
Once https://github.com/DataBiosphere/duos-ui/pull/1671 lands, pull those changes into this branch and we should be good 👍🏽

**Review by JVThomas — APPROVED** [review]

**Pull request merged**